changelog.md

```diff
@@ -1,3 +1,12 @@
+<a name="9.6.3"></a>
+# 9.6.3 (2022-06-28)
+[Full Changelog](https://github.com/compare/v9.6.2...v9.6.3)
```

Comment:
Add a note about this only going to npm

Reply:
"We have no plans to release this specific version to DevTools or PSI, just NPM"